pkg/tsdb/phlare/query.go

```diff
@@ -331,7 +380,7 @@ func treeToNestedSetDataFrame(tree *ProfileTree, profileTypeID string) *data.Fra
 		selfField.Append(tree.Self)
 		// todo: inline functions
 		// tree.Inlined
-		labelField.Append(tree.Function.FunctionName)
+		labelField.Append(tree.Function.String())
```

Comment:
Sounds like we're adding line and fileName twice using `String()` should we just take the Name here instead ?

Reply:
Will roll back this. Was there mainly for quick testing but I will do a separate front end PR to show the file and line numbers.